### DISCUSSION TITLE
[BETA] Userscripts v4.4.0 (macOS), v1.4.0 (iOS)

### DISCUSSION THREAD

**Discussion by quoid** [quoid/userscripts#375] [Announcements]
## Build 66/39
- [macOS/iOS] - refactor settings storage - #373
- [macOS/iOS] - misc style and code changes - #374

## Build 67/40
- [macOS] - refactor settings storage - #373
- [macOS/iOS] - misc style and code changes - #374, #376, #433, #442
- [macOS] - optimize `save location` interactive experience - #404
- [macOS/iOS] - update default file name patterns (`.user.js` suffix now added) - #436
- [macOS/iOS] - refactor script installation experience from `popup` - #439

**Comment by lindacuster** [comment]
Using 4.4.0 for Mac Safari (M1 Pro chip). Latest released macOS, latest released Safari. All working well. Thanks so much for wonderful software!

I'm seeing the </> icon in the Safari toolbar. I'm seeing a red badge with the number "1" there that is distracting me. What does it mean, and can I hide or suppress it?

**Reply by quoid** [reply]
That means you have 1 active userscript. You should be able to disable that from the settings page. However there is a bug that was preventing that setting from working and will be fixed in the next build (and it currently pushed to the beta build).

Issue: https://github.com/quoid/userscripts/issues/468

Screenshot of setting:
<img width="1171" alt="Screen Shot 2023-04-23 at 9 37 01 PM" src="https://user-images.githubusercontent.com/7660254/233880497-85569682-bca8-44f8-8efa-1d4ffcd7a808.png">

**Comment by lindacuster** [comment]
Incredible support! Thanks for being so responsive.

(I’d tried that Show Toolbar Count toggle and ran into the bug. Thanks for explaining you’re working on it.)
